website/templates/profile/oauth_app_list.mako

```diff
@@ -0,0 +1,71 @@
+<%inherit file="base.mako"/>
+<%def name="title()">OAuth Application settings</%def>
+<%def name="content()">
+<h2 class="page-header">OAuth application settings</h2>
+
+<div class="row">
+
+    <div class="col-sm-3">
+        <div class="panel panel-default">
+            <ul class="nav nav-stacked nav-pills">
+                <li><a href="${ web_url_for('user_profile') }">Profile Information</a></li>
+                <li><a href="${ web_url_for('user_account') }">Account Settings</a></li>
+                <li><a href="${ web_url_for('user_addons') }">Configure Add-ons</a></li>
+                <li><a href="${ web_url_for('user_notifications') }">Notifications</a></li>
+                <li><a href="#">Developer apps</a> </li>
+            </ul>
+        </div><!-- end sidebar -->
+    </div>
+
+    <div class="col-sm-9 col-md-7">
+        <a href="${web_url_for('oauth_application_register')}" role="button" class="btn btn-default pull-right"><i class="fa fa-plus"></i> Register new application</a>
+        <div id="app-list">
+
+            <p data-bind="visible: (content().length == 0)">You have not registered any applications that can connect to the OSF.</p>
```

Comment:
I totally agree that some concise, well-chosen help text would be useful. Most likely this would appear on the "Register new" page?

The sidebar navigation element link this page "Developer apps", but I wouldn't rely solely on the word "developer" to scare people away.